packages/smooth_ui_library/lib/buttons/smooth_simple_button.dart

```diff
@@ -14,7 +14,7 @@ class SmoothSimpleButton extends StatelessWidget {
     return Container(
       width: width,
       child: MaterialButton(
-        color: Theme.of(context).buttonColor,
+        color: Theme.of(context).buttonTheme.colorScheme.primary,
```

Comment:
My remarks:
* shouldn't the text be `onPrimary` too?
* is there a default theme for `MaterialButton` instead?